src/Hexagon/Adapter.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace DDDHH\Hexagon;
+
+abstract class Adapter {
+    abstract public function response($content);
+}
```

Comment:
I saw this _no newline at end of file_ hint in the `.gitignore` also. Most editors have a configuration setting to add this automatically.

Maybe we should add an [editorconfig](http://editorconfig.org/) to the project?

Besides that I don't see the point of this `Hexagon` thing. The abstraction is shaky. One could use **any implementation** of those `Port` and `Adapter` interfaces with the `ExternalCalculatorService` with unforeseeable effects. 

I see the point of abstracting the transport layer. But why not use an classic Anticorruption Layer? Define an `ExternalCalculatorServiceClient` interface with corresponding request and response interfaces to implement.

This way it would make testing easier also which you skipped completely in this PR.